Assets/UI/diplomacydealview_CQUI.lua

```diff
@@ -277,11 +277,11 @@ function CQUI_renderCityButton(pCity : table, player : table, targetContainer :
 
     if pCity:IsOccupied() then
         if pCity:GetOwner() == otherPlayer:GetID() then -- Cede
-            button.IconText:SetText(button.IconText:GetText() .. '[COLOR_Civ6Green] - ' .. Locale.Lookup("LOC_IDS_DEAL_CEDE") .. '[ENDCOLOR]'); 
+            button.IconText:SetText('[COLOR_Civ6Green]' .. Locale.Lookup("LOC_DIPLOMACY_DEAL_CEDE_CITY", cityData.CityName) .. '[ENDCOLOR]'); 
```

Comment:
Why this change?

Reply:
When I was looking at the text from IDS that was used in game, I noticed that like half the languages had the cede/return, half did not, and those that didn't would use English, I think. I then saw it's in the game anyways, and figured the trade off where the city name is now also colorized was okay (and I ended up preferring it anyway)

![image](https://user-images.githubusercontent.com/8787640/88362855-e60ea600-cd32-11ea-8495-4b727e251d7b.png)